e2e/tests/tabs-component.test.ts

```diff
@@ -38,5 +38,15 @@ test.describe('tabs-component test', async () => {
     const contentBText = await page.evaluate(node => node?.innerHTML, contentB);
     expect(tabBText).toContain('label2');
     expect(contentBText).toContain('content2');
+    const notSelected = await page.$$('.tabs-b div.not-selected_fa768');
```

Comment:
`.not-selected_fa768` is an unstable selector and may be broken in the future iterations.

Can we use another selector

Reply:
ok, theoretically hash should be stable.

I changed the way of selecting, pls check it again